static/bidder-info/impactify.yaml

```diff
@@ -0,0 +1,13 @@
+maintainer:
+  email: "thomas.destefano@impactify.io"
```

Comment:
Please use a group email address to protect against individual roles or responsibilities changing over time.

Reply:
Done.